modules/health_quest/spec/serializers/v0/va_appointments_serializer_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe HealthQuest::V0::VAAppointmentsSerializer do
+  subject { described_class.new(appt[:data], meta: appt[:meta]) }
+
+  let(:klass) { described_class }
+  let(:user) { build(:user, :health_quest) }
+  let(:appt_service) { HealthQuest::AppointmentService.new(user) }
+  let(:appt) { appt_service.get_appointment_by_id(132) }
+
+  describe 'subject type' do
+    it 'is a HealthQuest::V0::VAAppointmentsSerializer' do
+      expect(subject).to be_a(HealthQuest::V0::VAAppointmentsSerializer)
+    end
+  end
+
+  describe 'included' do
+    it 'includes FastJsonapi::ObjectSerializer' do
+      expect(klass).to include(FastJsonapi::ObjectSerializer)
+    end
+  end
+
+  describe 'KEY #meta' do
+    it 'has meta key' do
+      expect(subject.serializable_hash.key?(:meta)).to be(true)
+    end
+
+    it 'has default values' do
+      expect(subject.serializable_hash[:meta]).to eq({ 'pagination': {
+                                                       'current_page': 0,
+                                                       'per_page': 0,
+                                                       'total_pages': 0,
+                                                       'total_entries': 0
+                                                     } })
```

Comment:
```suggestion
      expect(subject.serializable_hash[:meta]).to eq({
        'pagination': {
          'current_page': 0,
          'per_page': 0,
          'total_pages': 0,
          'total_entries': 0
        }
      })
```

What do you think of this syntax?

Reply:
Yes, that's definitely better. I will go with your recommendation. I let rubocop autocorrect my version and didn't realize it looked this ghastly.

Reply:
Rubocop still won't let me align it the way you suggested. I've provided a workaround. Let me know if that works for you. Thanks!

Reply:
Oh. I think rubocop wants a new line between `(` and `{`. Your solution works fine though. :+1: